project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.hanhuy.sbt" % "android-sdk-plugin" % "1.5.19")
+addSbtPlugin("org.scala-android" % "sbt-android" % "1.6.17")
```

Comment:
I have tested the app in my local repository in a simple project and works fine. What problem can we have?